base_controller:
  pid_gains:
      wheel_joint:
          p : 50
          i : 10

  # Velocity and acceleration limits
  # Whenever a min_* is unspecified, default to -max_*
  linear:
    x:
      has_velocity_limits    : true
      max_velocity           : 1.8  # m/s
      min_velocity           : -1.8 # m/s
      has_acceleration_limits: true
      max_acceleration       : 1.2  # m/s^2
      min_acceleration       : -1.2 # m/s^2
      has_jerk_limits        : true
      max_jerk               : 3.8  # m/s^3
  angular:
    z:
      has_velocity_limits    : true
      max_velocity           : 8.0  # rad/s
      min_velocity           : -8.0  # rad/s
      has_acceleration_limits: true
      max_acceleration       : 6.8  # rad/s^2
      has_jerk_limits        : true
      max_jerk               : 6.5  # rad/s^3
